### PR TITLE
Add --prune to clean up dependancy folders after install

### PR DESCRIPTION
**Pull request by andrewkdouglas**
Just an idea...

### CODE REVIEW

**Comment by oNaiPs** [comment]
Great idea!
Some comments before merging:

Could you put your README.md changes in markdown format instead?

Could you make the spawn + return code check into a function to skip the redundant code for both calls?

Thanks!

**Comment by andrewkdouglas** [comment]
Sure done!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "npm-install-changed",
-  "version": "1.2.1",
+  "version": "1.3.5",
```
Comment (oNaiPs) [inline package.json]: Should we bump to a more sane number?

Comment (andrewkdouglas) [reply package.json]: Sure, 1.3.5 is my internal one. 1.3.0?

Comment (oNaiPs) [reply package.json]: Sounds good!

Comment (oNaiPs) [reply package.json]: @adouglas could you remove the version from your PR? I will bump the version after merging.

**Comment by andrewkdouglas** [comment]
@oNaiPs Version number removed, sorry for the delay.

**Pull request merged**